sdk/internal/list/attr.go

```diff
@@ -0,0 +1,251 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
I don't know that we can/should claim this.  Do we have a policy/process for handling imported derivative works like this?

Reply:
Looking at [the original license](https://github.com/golang/go/blob/master/LICENSE) I think we can add this copyright as it would copyright the modification of the original, but we will need to include the original copyright below. I'll add a commit doing this.